### DISCUSSION TITLE
Workflow example with VS Code integration

### DISCUSSION THREAD

**Discussion by gee4vee** [eclipse-glsp/glsp#675] [Q&A]
Is there a [workflow example](https://github.com/eclipse-glsp/glsp-examples/tree/master/workflow) with VS Code integration using a node server? Or would it be possible to use with the VS Code integration code from the task list example? Or to bring in the workflow model into the [task list example](https://github.com/eclipse-glsp/glsp-examples/tree/master/project-templates/node-json-vscode)?

**Comment by planger** [comment]
Sorry, for the late reply!

No, afaik there isn't a workflow example with VS Code integration using a node server. The bits and pieces exist, though.

* Workflow diagram server based on node: https://github.com/eclipse-glsp/glsp-server-node/tree/main/examples/workflow-server
* VS Code integration of the workflow diagram: https://github.com/eclipse-glsp/glsp-vscode-integration/tree/master/example/workflow
* Project template for integrating a node-based server in a VS Code extension: https://github.com/eclipse-glsp/glsp-examples/tree/master/project-templates/node-json-vscode/glsp-client

So I think those pieces need to be combined in order to achieve what you need.

For future reference: There is ongoing discussion related to this in https://github.com/eclipse-glsp/glsp/discussions/681
